test/example_data.dart

```diff
@@ -278,6 +299,7 @@ Map<String, dynamic> _messagePropertiesFromContent(String? content, String? cont
 /// A fresh message ID, from a random but always strictly increasing sequence.
 int _nextMessageId() => (_lastMessageId += 1 + Random().nextInt(100));
 int _lastMessageId = 1000;
+const defaultStreamMessageStreamId = 123;
```

Comment:
nit:
```suggestion
int _nextMessageId() => (_lastMessageId += 1 + Random().nextInt(100));
int _lastMessageId = 1000;

const defaultStreamMessageStreamId = 123;
```

The first two items are closely related to each other, but not to the third. So the visual grouping can reflect that.